config/webpack.target.browser.js

```diff
@@ -32,7 +35,10 @@ module.exports = function(production, app) {
   if (
     fs.existsSync(path.resolve(__dirname, `../targets/${app}/web/services.jsx`))
   ) {
-    entry.services = [path.resolve(__dirname, 'expose-react.js'), path.resolve(__dirname, `../targets/${app}/web/services`)]
+    entry.services = [
+      path.resolve(__dirname, 'expose-react.js'),
```

Comment:
I think that we don't need React for services

Reply:
We shouldn't, but we do — somehow components that use `BarLeft` etc are imported on the services view.